### PR TITLE
update antsibull-docs to 1.7.3

### PR DESCRIPTION
**Pull request by samccann**
##### SUMMARY

To support Ansible 7 release.  We recognize that the CI containers will be out of sync with minor changes to `ansibull-docs`.
ansibull-docs 1.7.3 fixes the following:

- Fix rendering of the action_group attribute (https://github.com/ansible-community/antsibull-docs/pull/62).
- Fix version_added processing for ansible.builtin 0.x to represent this as Ansible 0.x instead of ansible-core 0.x (https://github.com/ansible-community/antsibull-docs/pull/61).

##### ISSUE TYPE

- Bugfix Pull Request
- Docs Pull Request

##### COMPONENT NAME

test/sanity/code-smell/docs-build.requirements.in
##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

File: test/sanity/code-smell/docs-build.requirements.txt [hunk 1]
```diff
@@ -1,19 +1,19 @@
...
 aiofiles==22.1.0
 aiohttp==3.8.3
-aiosignal==1.2.0
+aiosignal==1.3.1
 alabaster==0.7.12
 ansible-pygments==0.1.1
-antsibull-core==1.2.0
-antsibull-docs==1.7.0
+antsibull-core==1.3.1
+antsibull-docs==1.7.3
```
Comment (felixfontein) [inline test/sanity/code-smell/docs-build.requirements.txt:8]: I would undo all changes except the one in line 8.

**Comment by felixfontein** [comment]
From my POV it is also fine if 1.7.3 is only used during the real docs build and not during the docs sanity test.

**Review by felixfontein — APPROVED** [review]

**Comment by samccann** [comment]
The docs build for latest actually works off the CI test requirements file so needs this change if we don't want to be hacking the jenkins jobs themselves for this.

/me doesn't want to be hacking the jenkins jobs themselves for this...

**Comment by samccann** [comment]
Verified on test - http://docs.testing.ansible.com/ansible/devel/collections/community/docker/docker_container_module.html#attributes

This will need to be backported to stable-2.14

**Comment by samccann** [comment]
See https://github.com/ansible/ansible/pull/79406#issuecomment-1319171634 for notes on how to handle this in the stable-branch.

**Pull request merged**